Jack/Gen.fs

```diff
@@ -434,6 +434,15 @@ module Gen =
     let float : Gen<float> =
         double |> map float
 
+    let dateTime : Gen<System.DateTime> =
+        gen { let! y = range 1753 9999
```

Comment:
Would it ever make sense to have a larger range? The gregorian calendar is reasonable from about 1600 onwards for example.